tests/cypress/support/commands.ts

```diff
@@ -373,3 +384,39 @@ Cypress.Commands.add('deleteRole', (roleName, roleTypeTemplate) => {
   cy.confirmDelete();
   cy.contains('There are no rows which match your search query.', { timeout: 2000 }).should('be.visible');
 })
+
+// Add label to the imported cluster(s)
+Cypress.Commands.add('assignClusterLabel', (clusterName, key, value) => {
+  cy.accesMenuSelection('Continuous Delivery', 'Clusters');
+  cy.contains('.title', 'Clusters').should('be.visible');
+  cy.filterInSearchBox(clusterName);
+  cy.open3dotsMenu(clusterName, 'Edit Config');
+  cy.clickButton('Add Label');
+  cy.get('input[placeholder="e.g. foo"]').type(key);
```

Comment:
hint maybe going in new direction?
```
cy.get('div[class='row'] div[class='key-value'] input[placeholder="e.g. foo"]').last().type(key) ?
```

Reply:
I have updated with your suggestions, Thanks :smile: